### PR TITLE
[3558] Single file upload is not handling errors.

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/3558

### CODE REVIEW

File: static-assets/components/cstudio-dialogs/upload-asset-dialog.js [hunk 1]
```diff
@@ -180,6 +183,28 @@ CStudioAuthoring.Dialogs.UploadDialog = CStudioAuthoring.Dialogs.UploadDialog ||
...
+
+                me.uploadingFile = false;
+                $('#uploadCancelButton').attr('disabled', false);
+
+                CStudioAuthoring.Operations.showSimpleDialog(
+                  "uploadErrorDialog",
+                  CStudioAuthoring.Operations.simpleDialogTypeINFO,
+                  me.formatMessage(me.messages.notification),
+                  errorMsg,
+                  [{ text: "OK",  handler:function(){
```
Comment (rart) [inline static-assets/components/cstudio-dialogs/upload-asset-dialog.js]: Fix formatting here please

Single obj
```
[{
 prop: value,
 prop: value
}]
```

Multiple obj
```
[
  {
   prop: value,
   prop: value
  },
  {
   prop: value,
   prop: value
  }
]
```

File: static-assets/components/cstudio-dialogs/uploadS3-dialog.js [hunk 2]
```diff
@@ -179,6 +182,28 @@ CStudioAuthoring.Dialogs.UploadS3Dialog = CStudioAuthoring.Dialogs.UploadS3Dialo
...
+
+          me.uploadingFile = false;
+          $('#uploadCancelButton').attr('disabled', false);
+
+          CStudioAuthoring.Operations.showSimpleDialog(
+            "uploadErrorDialog",
+            CStudioAuthoring.Operations.simpleDialogTypeINFO,
+            me.formatMessage(me.messages.notification),
+            errorMsg,
+            [{ text: "OK",  handler:function(){
```
Comment (rart) [inline static-assets/components/cstudio-dialogs/uploadS3-dialog.js]: Format as requested

File: static-assets/components/cstudio-dialogs/uploadWebDAV-dialog.js [hunk 3]
```diff
@@ -164,6 +167,28 @@ CStudioAuthoring.Dialogs.UploadWebDAVDialog = CStudioAuthoring.Dialogs.UploadWeb
...
           CStudioAuthoring.Dialogs.UploadWebDAVDialog.closeDialog();
+        },
+        onError: function(file, error, response) {
+          const res = response.body.response,
+            errorMsg = `${res.message}. ${res.remedialAction}`;
+
+          me.uploadingFile = false;
+          $('#uploadCancelButton').attr('disabled', false);
+
+          CStudioAuthoring.Operations.showSimpleDialog(
```
Comment (rart) [inline static-assets/components/cstudio-dialogs/uploadWebDAV-dialog.js:178]: Format as requested

**Review by rart — CHANGES_REQUESTED** [review]
Please also reformat where ever you copied snippet from

File: ui/app/src/utils/i18n-legacy.ts [hunk 4]
```diff
@@ -142,6 +142,13 @@ const contentTypesMessages = defineMessages({
 
 });
 
+const commonMessages = defineMessages({
+  notification: {
+    id: 'commonMessages.notification',
+    defaultMessage: 'Notification'
+  }
+});
+
```
Comment (rart) [inline ui/app/src/utils/i18n-legacy.ts]: Let's rename this to words. Only single words may go in here.

```js
const words = defineMessages(
  ...
  id: 'words.notification'
  ...
)
```

Carry this over to the history PR (and others as required).

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/utils/i18n-legacy.ts (hunk 4 above)
Comment (jvega190) [reply ui/app/src/utils/i18n-legacy.ts]: done, history PR will be updated as well

**Review by rart — APPROVED** [review]

**Pull request merged**